src/servers/graphql-main-server.ts

```diff
@@ -17,6 +17,7 @@ export async function startApolloServerForCoreSchema() {
     {
       Query: {
         me: isAuthenticated,
+        onChainAddressCurrent: isAuthenticated,
```

Comment:
Shouldn't this be scoped under `wallet`?
Like: `me.defaultAccount.wallet.onChainAddressCurrent`? cc @samerbuna

Reply:
Move to mutation

Reply:
fixed